packages/mui-material/src/Autocomplete/Autocomplete.d.ts

```diff
@@ -237,13 +237,15 @@ export interface AutocompleteProps<
    *
    * @param {object} props The props to apply on the li element.
    * @param {T} option The option to render.
-   * @param {object} state The state of the component.
+   * @param {object} state The state of each option.
+   * @param {object} ownerState The state of the component and all the props it is given to.
```

Comment:
Now this is actually the state of the component:
```suggestion
   * @param {object} ownerState The state of the Autocomplete component.
```